File: Entities/Devolucion.cs
namespace Entidades
{
    using System;

    public class Devolucion
    {
        private int _varIdDevolucion;
        private int _varIdProveedor;
        private string _varFactura;
        private DateTime _varFecha;
        private string _varBoleta;
        private double _varCantidad;
        private string _varProducto;
        private int _varCodigo;
        private string _varMotivo;
        private string _varObservaciones;

        public int IdDevolucion
        {
            get => 
                this._varIdDevolucion;
            set => 
                this._varIdDevolucion = value;
        }

        public int IdProveedor
        {
            get => 
                this._varIdProveedor;
            set => 
                this._varIdProveedor = value;
        }

        public string Factura
        {
            get => 
                this._varFactura;
            set => 
                this._varFactura = value;
        }

        public DateTime Fecha
        {
            get => 
                this._varFecha;
            set => 
                this._varFecha = value;
        }

        public string Boleta
        {
            get => 
                this._varBoleta;
            set => 
                this._varBoleta = value;
        }

        public double Cantidad
        {
            get => 
                this._varCantidad;
            set => 
                this._varCantidad = value;
        }

        public string Producto
        {
            get => 
                this._varProducto;
            set => 
                this._varProducto = value;
        }

        public int Codigo
        {
            get => 
                this._varCodigo;
            set => 
                this._varCodigo = value;
        }

        public string Motivo
        {
            get => 
                this._varMotivo;
            set => 
                this._varMotivo = value;
        }

        public string Observaciones
        {
            get => 
                this._varObservaciones;
            set => 
                this._varObservaciones = value;
        }
    }
}

File: Entidades/Devolucion.vb
Public Class Devolucion

    Private _varIdDevolucion As Int32
    Public Property IdDevolucion() As Int32
        Get
            Return _varIdDevolucion
        End Get
        Set(ByVal value As Int32)
            _varIdDevolucion = value
        End Set
    End Property

    Private _varIdProveedor As Int32
    Public Property IdProveedor() As Int32
        Get
            Return _varIdProveedor
        End Get
        Set(ByVal value As Int32)
            _varIdProveedor = value
        End Set
    End Property

    Private _varFactura As String
    Public Property Factura() As String
        Get
            Return _varFactura
        End Get
        Set(ByVal value As String)
            _varFactura = value
        End Set
    End Property

    Private _varFecha As Date
    Public Property Fecha() As Date
        Get
            Return _varFecha
        End Get
        Set(ByVal value As Date)
            _varFecha = value
        End Set
    End Property

    Private _varBoleta As String
    Public Property Boleta() As String
        Get
            Return _varBoleta
        End Get
        Set(ByVal value As String)
            _varBoleta = value
        End Set
    End Property

    Private _varCantidad As Double
    Public Property Cantidad() As Double
        Get
            Return _varCantidad
        End Get
        Set(ByVal value As Double)
            _varCantidad = value
        End Set
    End Property

    Private _varProducto As String
    Public Property Producto() As String
        Get
            Return _varProducto
        End Get
        Set(ByVal value As String)
            _varProducto = value
        End Set
    End Property

    Private _varCodigo As Int32
    Public Property Codigo() As Int32
        Get
            Return _varCodigo
        End Get
        Set(ByVal value As Int32)
            _varCodigo = value
        End Set
    End Property

    Private _varMotivo As String
    Public Property Motivo() As String
        Get
            Return _varMotivo
        End Get
        Set(ByVal value As String)
            _varMotivo = value
        End Set
    End Property

    Private _varObservaciones As String
    Public Property Observaciones() As String
        Get
            Return _varObservaciones
        End Get
        Set(ByVal value As String)
            _varObservaciones = value
        End Set
    End Property
End Class
